stubs/inertia-react/resources/js/Components/TextInput.jsx

```diff
@@ -17,6 +17,7 @@ export default forwardRef(function TextInput(
             <input
                 type={type}
                 name={name}
+                id={name}
```

Comment:
I'm not a fan of re-using the `name` prop here as sometimes they need to be different.

I think it should be `id={id}` with a new `id` prop added to the component definition and everywhere the component is used with a label.

Reply:
I have add id in all component used TextInput.jsx